Formula/sk.rb

```diff
@@ -0,0 +1,25 @@
+class Sk < Formula
+  desc "Fuzzy Finder in rust!"
+  homepage "https://github.com/lotabout/skim"
+  url "https://github.com/lotabout/skim/archive/v0.6.6.tar.gz"
+  sha256 "5975b21a7c3792e910ee9439b41b417e92f2fc7bc92d033e2d4c6d6c48a469c8"
+  head "https://github.com/lotabout/skim.git"
+
+  depends_on "rust" => :build
+
+  def install
+    (buildpath/"src/github.com/lotabout").mkpath
+    ln_s buildpath, buildpath/"src/github.com/lotabout/skim"
+    system "cargo", "install", "--root", prefix, "--path", "."
+
+    pkgshare.install "install"
+    bash_completion.install %w[bash zsh].map { |s| "shell/key-bindings.#{s}" }
```

Comment:
This doesn't look right; you're installing zsh completions with `bash_completion.install`. Please remove the `map` and just hardcode the values, thanks.